server/utils/routeUtils.ts

```diff
@@ -0,0 +1,9 @@
+import type { RequestHandler, Router } from 'express'
+
+import asyncMiddleware from '../middleware/asyncMiddleware'
+
+export default function actions(router: Router) {
+  return {
+    get: (path: string | Array<string>, handler: RequestHandler) => router.get(path, asyncMiddleware(handler)),
+  }
```

Comment:
In Approved Premises this was in `server/routes/utils.ts` and didn't have an accompanying unit test. I put it in the main `utils` folder like we've done with `typeUtils`. The method in `typeUtils` in our repo doesn't have a unit test either, but I added a card for doing this (if we want it). Shall we add unit tests for both?

Reply:
Yeah, it'd be good to have a unit test for this in this PR, and we can add another one for that other method when we pick up that technical debt card.

Reply:
To be completed separately, since the mocking is a little complicated